srcs/executing/executor.c

```diff
@@ -43,6 +39,22 @@ static void	close_for_next_cmd(t_cmd_table cmd)
 		close(cmd.fd_out);
 }
 
+// child process:
+// close pipe's reading end? Which is in next cmd fd_in?
+// if (fd_in != 0)
+// {
+// 	dup2(fd_in, STDIN_FILENO);
+// 	close(fd_in);
+// }
+// if (fd_out != 1)
+// {
+// 	dup2(fd_out, STDOUT_FILENO);
+// 	close(fd_out);
+// }
+// if (valid_command(pipex, 1)) // look into this function
+// 	execve(pipex->cmd_path, pipex->command, pipex->envp); // look into pipex struct
+// error_message(WRONG_CMD); // need to add this
+
```

Comment:
We'll need to brainstorm on the whole multi pipe thing. Because we need to go from scratch for pipex from what I understand. 
Maybe we can talk with some people about it once we have some ideas on how to deal with it ^^

Reply:
I was talking with Maria and José, they made me see something I wasn't counting with. I think I have an idea, just need to wrap my head around it and implement it.

Reply:
oki doke